### PR TITLE
Release 1.2.1

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This upgrades us to Apollos Project Core version `1.2.1`.

## TODO

- [x] I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-church-api/config.yml [hunk 1]
```diff
@@ -50,6 +52,7 @@ ANALYTICS:
   # on the accompanying service
   SEGMENT_KEY:
   GA_ID:
+  USE_ROCK:
```
Comment (redreceipt) [inline packages/apollos-church-api/config.yml]: should this pull from our `.env` file? we should match however core does it

Comment (richarddubay) [reply packages/apollos-church-api/config.yml]: @redreceipt Yep, the one that came in from core in the update just said `USE_ROCK: true`. I'll make it `false`.

Comment (richarddubay) [inline packages/apollos-church-api/config.yml]:
```suggestion
  USE_ROCK: false
```

File: packages/newspringchurchapp/src/ui/ContentCardConnected/query.js [hunk 2]
```diff
@@ -33,23 +33,23 @@ export const BASE_CARD_FRAGMENT = gql`
...
     ... on WeekendContentItem {
       videos {
         sources {
           uri
         }
       }
-      liveStream {
-        isLive
-      }
       parentChannel {
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/ui/ContentCardConnected/query.js:53]: probably should delete it from down here since core added it above

Comment (richarddubay) [reply packages/newspringchurchapp/src/ui/ContentCardConnected/query.js:53]: I think `parentChannel` is on each of the "content items" now. https://github.com/ApollosProject/apollos-prototype/blob/master/packages/apolloschurchapp/src/ui/ContentCardConnected/query.js#L53

**Review by redreceipt — APPROVED** [review]

**Pull request merged**